Jellyfin.Api/Controllers/AudioController.cs

```diff
@@ -91,7 +91,7 @@ public AudioController(AudioHelper audioHelper)
         [ProducesResponseType(StatusCodes.Status200OK)]
         public async Task<ActionResult> GetAudioStream(
             [FromRoute, Required] Guid itemId,
-            [FromRoute, Required] string? container,
+            [FromRoute] string? container,
```

Comment:
It doesn't validate the input, but you can use ` {itemId}/stream.{container:required}` if you want container to be required when matching that particular route